sdk/core/azure-core/src/main/java/com/azure/core/util/FluxUtil.java

```diff
@@ -103,6 +103,34 @@ private static void accept(ByteArrayOutputStream byteOutputStream, ByteBuffer by
             .flatMap(serviceCall);
     }
 
+    /**
+     * This method converts the incoming {@code subscriberContext} from {@link reactor.util.context.Context Reactor
+     * Context} to {@link Context Azure Context}, adds the specified tracing attributes and calls the given lambda
+     * function with this context and returns a single entity of type {@code T}
+     * <p>
+     * If the reactor context is empty, {@link Context#NONE} will be used to call the lambda function
+     * </p>
+     *
+     * @param serviceCall serviceCall The lambda function that makes the service call into which azure context
+     * will be passed
+     * @param contextAttributes The map of attributes to be set on the tracer spans.
```

Comment:
The map of attributes doesn't necessarily have to be for tracer spans. Update javadoc here to just say  `The map of attributes to be set on {@link Context}`